test/functional/wallet_dump.py

```diff
@@ -73,7 +102,7 @@ def run_test (self):
             read_dump(tmpdir + "/node0/wallet.unencrypted.dump", addrs, None)
```

Comment:
even when not directly set in this PR, it's a good moment to fix this paths here to. there's a few instances in this PR.
We should replace these strings using `os.path.join`

Reply:
done